packages/api/src/provider-info.ts

```
@@ -157,3 +157,27 @@ export interface ProviderCleanupActionInfo {
+  connectionType: 'kubernetes';
+  // ...existing fields
+}
+
+export interface ProviderVmConnectionInfo {
+  connectionType: 'vm';
+  name: string;
+  status: ProviderConnectionStatus;
+  lifecycleMethods?: LifecycleMethod[];
+}
+
+export type ProviderConnectionInfo =
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ProviderConnectionInfo'. Consider to delete it or rename it.

'ProviderConnectionInfo' is defined here:


(lint/suspicious/noRedeclare)